automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -261,6 +261,33 @@ class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with W
 
 
   "With dockstore method config" - {
+    "defaults to run without entity model" in {
+      // written for GAWB-3695
+      val user = UserPool.chooseProjectOwner
+      implicit val authToken: AuthToken = user.makeAuthToken()
+
+      withCleanBillingProject(user) { projectName =>
+
+        withWorkspace(projectName, "MethodConfigSpec", attributes = Some(Map("foo" -> "bar"))) { workspaceName =>
+          // Create method config
+          val configName = s"test_dockstore_config_no_model$workspaceName"
+          api.methodConfigurations.createDockstoreMethodConfigInWorkspace(
+            projectName, workspaceName, DockstoreMethodData.dockstoreMethod,
+            projectName, configName)
+
+          withWebDriver { implicit driver =>
+            withSignIn(user) { _ =>
+              // Go to method config page
+              val configPage = new WorkspaceMethodConfigDetailsPage(projectName, workspaceName, projectName, configName).open
+
+              // We should not be using the entity data model
+              configPage.isUsingDataModel() shouldBe false
+
+            }
+          }
+        }
+      }
+    }
```

Comment:
I'm fine with this being a separate test, but the one right below it is doing mostly the same thing - creating a workspace, and importing a methodConfig and then checking 1 thing in that method config. Part of me feels like both these tests should actually be one test - that making them separate just increases the amount of time it takes for the whole suite to pass. 

I'm not advocating a change - just voicing a concern about how long it takes for us to run tests and how that could be decreased if we consolidated our tests more.